-DIUTEST_HAS_STRINGSTREAM=0 test faild

```
../include/gtest/../iutest_any.hpp:250:15: error: request for member ‘to_string’ in ‘value’, which is of non-class type ‘const int* const’
return os << value.to_string();
```

--- a/include/iutest_any.hpp
+++ b/include/iutest_any.hpp
@@ -239,13 +239,12 @@ inline T unsafe_any_cast(const any& value)
 
 #if IUTEST_HAS_STRINGSTREAM || IUTEST_HAS_STRSTREAM
 template<typename Elem, typename Traits>
-::std::basic_ostream<Elem, Traits>& operator << (::std::basic_ostream<Elem, Traits>& os, const any& value)
+inline ::std::basic_ostream<Elem, Traits>& operator << (::std::basic_ostream<Elem, Traits>& os, const any& value)
 {
     return os << value.to_string();
 }
 #else
-template<typename T>
-iu_ostream& operator << (iu_ostream& os, const T& value)
+inline iu_ostream& operator << (iu_ostream& os, const any& value)
 {
     return os << value.to_string();
 }

--- a/test/printers_tests.cpp
+++ b/test/printers_tests.cpp
@@ -67,8 +67,8 @@ IUTEST(PrintToTest, IutestAnyNotInitialized)
 
 IUTEST(PrintToTest, IutestAnyString)
 {
-    ::iutest::any a = "test";
-    LogChecker ck("test");
+    ::iutest::any a = "any-test";
+    LogChecker ck("any-test");
     IUTEST_SUCCEED() << ::iutest::PrintToString(a);
 }
 

